### PR TITLE
Add new tooltips, no results found text when 0 results

### PR DESCRIPTION
**Pull request by TimJones97**
- Banner tooltip highlights unrealised profits shown as portfolio value
- Portfolio value table header tooltip highlights pending mints and burns not included in value

### CODE REVIEW

**Comment by dospore** [comment]
Soz it snuck through last PR but we I pref yarn.lock over package-lock, the two lock files cause a few issues so only commit 1

**Comment by TimJones97** [comment]
> Soz it snuck through last PR but we I pref yarn.lock over package-lock, the two lock files cause a few issues so only commit 1

No problems, it's removed here but will remember to use yarn only for future

File: components/Nav/Navbar/MobileMenu/index.tsx [hunk 1]
```diff
@@ -132,6 +132,20 @@ export default (({ account, className }) => {
...
+                                            )}
+                                            onClick={() => handleRoute('/trading-comp')}
+                                        >
+                                            <img
+                                                className="mr-2 inline"
+                                                src={'/img/general/trading-comp.svg'}
+                                                alt="Trading Comp"
+                                            />
+                                            Trading Comp
+                                        </div>
```
Comment (dospore) [inline components/Nav/Navbar/MobileMenu/index.tsx:148]: do you mind quickly aligning this with the other links as well, remove the stake page and stuff

**Review by dospore — APPROVED** [review]
Looks good, happy to merge in if we want to update mobile links in a seperate PR

**Review by dospore — APPROVED** [review]

**Pull request merged**